sensor/common/registry/registry_store_test.go

```diff
@@ -35,194 +42,310 @@ func alwaysInsecureCheckTLS(_ context.Context, _ string) (bool, error) {
 	return false, nil
 }
 
+// alwaysSecureCheckTLS is an implementation of registry.CheckTLS
+// which always says the given address is secure.
 func alwaysSecureCheckTLS(_ context.Context, _ string) (bool, error) {
 	return true, nil
 }
 
+// alwaysFailCheckTLS is an implementation of registry.CheckTLS
+// which always says there was an error checking TLS.
 func alwaysFailCheckTLS(_ context.Context, _ string) (bool, error) {
 	return false, errors.New("fake tls failure")
 }
 
-func TestRegistryStore_same_namespace(t *testing.T) {
-	ctx := context.Background()
+// TestRegistryStore_PullSecrets tests that the expected pull secrets
+// are retrieved (or not retrieved) from the store for various scenarios.
+func TestRegistryStore_PullSecrets(t *testing.T) {
+	dceA := config.DockerConfigEntry{Username: "usernameA", Password: "passwordA"}
+	dceB := config.DockerConfigEntry{Username: "usernameB", Password: "passwordB"}
+	dcA := config.DockerConfig{fakeImgName.GetRegistry(): dceA}
+	dcB := config.DockerConfig{fakeImgName.GetRegistry(): dceB}
+
+	// commonTests executes tests that should have the same results regardless of
+	// how secrets are stored.
+	commonTests := func(t *testing.T) {
+		namespace := "qa"
+
+		// Ensures that secrets are retrieved as expected.
+		t.Run("ensure expected secrets are retrieved", func(t *testing.T) {
+			regStore := NewRegistryStore(alwaysInsecureCheckTLS)
+
+			dce := config.DockerConfigEntry{Username: "username", Password: "password"}
+			dc := config.DockerConfig{
+				"image-registry.openshift-image-registry.svc:5000":       dce,
+				"image-registry.openshift-image-registry.svc.local:5000": dce,
+				"172.99.12.11:5000": dce,
+				"quay.io":           dce,
```

Comment:
I thought this would fail style checks

Reply:
The indentation/spacing? Since this is the output of `gofmt` perhaps OK per style check.